.env

```diff
@@ -32,6 +32,7 @@ FLAGSMITH_WEBHOOK_SECRET=webhook
 
 INTERNAL_FEED=http://localhost:6000/feed.json?token=token
 
+POST_SCRAPER_ORIGIN=http://localhost:8000
```

Comment:
Port `8000` seems to be available. Let me know if you would want something else.